sources/packages/web/src/assets/css/base.scss

```diff
@@ -3,11 +3,13 @@ html * {
   font-family: "BCSans", "Noto Sans", Verdana, Arial, sans-serif, "primeicons";
   font-style: $font-style-normal;
   line-height: 20px;
-  letter-spacing: 0px;
+  // TODO: commenting below css, since its overriding the letter spacing of class .nav-subtitle.
```

Comment:
IMO Instead of adding this in TODO, we can add this to the PR description so that whenever we need, we can lookup if anything breaks because of this change.

Reply:
okay